talli-app/src/components/OrganizerView/AddEntryOrg.js

```diff
@@ -1,17 +1,56 @@
 import React from 'react';
 import { Typography } from '@material-ui/core';
+import NewEntryForm from './NewEntryForm';
 import '../component_style/Organizer.css';
 
 /**
  * Entry Add/Remove, unimplemented
  * TODO: read existing events from database and render
  */
 export default class AddEntryOrg extends React.Component {
+    state = {
+        entries: [
+            {
+                title: '',
+                id: '',
+                presenters: '',
+                dates: ''
+            }
+        ],
+    };
+
+    addEntry = () => {
+        const newEntries = this.state.entries.slice();
+        newEntries.push({
+            title: '',
+            id: '',
+            presenters: '',
+            dates: ''
+        });
+        this.setState({ entries: newEntries });
+    }
 
     render() {
-        return(
-            <div>
+        let {entries} = this.state;
+        return (
+            <div className='addEntryForm'>
                 <Typography variant='h4' align='center' gutterBottom>Add Entries</Typography>
+                <button className='buttons' name='import_entries'>Import Entries</button>

```

Comment:
There doesn't appear to be a 'buttons' class in Organizer.css (the linked stylesheet).  Are you trying to use the 'buttons' class from MainPage.css?
Also I think it would be good (here and other places) to replace a lot of the builtin jsx tags with the equivalent Material-UI components (ie this would change to <Button>, etc) to match the rest of the app and have a more cohesive feel.  (I don't mind going through and fixing this if you want)

Reply:
Ohhh so that's where the Button tags are coming from, yeah definitely make those changes if you want